NOTICE

```diff
@@ -13,10 +14,14 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
-Most libraries in the following directories made by other people and
-organizations and licensed in different ways:
-* `3party`
-* `tools`
-Please refer to their LICENCE, COPYING or NOTICE files for terms of use.
-Some icons files may be copyrighted by (C) 2020 My.com B.V. (Mail.Ru Group)
-See also the `.reuse/dep5` and `data/copyright.html` file for a full list of copyright notices.
+Third-party dependencies are governed by separate licenses. Please see
+the `.reuse/dep5`, `data/copyright.html`, and individual `LICENSE`,
+`COPYING`, or `NOTICE` files for more details.
+
+Map data is from OpenStreetMap, licensed under the Open Data Commons Open
+Database License (ODbL) 1.0. You may obtain a copy of the Open Database
+License (ODbL) 1.0 at
+
+    https://opendatacommons.org/licenses/odbl/1-0/
+
+For any legal inquiries, feel free to contact legal@organicmaps.app.
```

Comment:
```suggestion
If you intend to use this code in your products, please follow the APACHE 2.0 license terms, and clearly state the copyright notice/link to the Organic Maps project, visible to all users of your derived products.

If you intend to use Organic Maps web servers (CDN) in your products, please contact legal@organicmaps.app

For any legal inquiries, feel free to contact legal@organicmaps.app.
```

Reply:
This statement is excessive here.

Reply:
> If you intend to use Organic Maps web servers (CDN) in your products, please contact legal@organicmaps.app

It’s not related to copyright or the Apache 2.0 NOTICE file.

Reply:
Yeah 

> please follow the APACHE 2.0 license terms, and clearly state the copyright notice/link to the Organic Maps project, visible to all users of your derived product

is ironically not in itself following the Apache license. People following the Apache license will naturally ensure that:

- Recipients of the Work get a copy of the Apache License
- There are notices stating if files are changed
- All copyright, patent, trademark, and attribution notices in the Source are retained